runtime/common/metering.go

```diff
@@ -145,10 +153,17 @@ func NewDictionaryMemoryUsages(length int) (MemoryUsage, MemoryUsage) {
 		}
 }
 
-func NewDictionarySizeUsage(length int) MemoryUsage {
+func NewDictionaryAdditionalSizeUsage(originalSize, additionalSize int) MemoryUsage {
+	var newAmount uint64
```

Comment:
The code here is a duplicate of the above, maybe factor the logic out. We might forget to update one of the functions